automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainSpec.scala

```diff
@@ -61,7 +61,7 @@ class AuthDomainSpec extends FlatSpec with Matchers with CleanUp with BillingFix
     withGroup("ad", List(projectUser.email, projectOwner.email)) { realmGroup =>
       withGroup("ad2", List(projectUser.email, projectOwner.email)) { realmGroup2 =>
         val realmGroup2Full = Orchestration.groups.getGroup(realmGroup2)(groupOwnerToken)
-        withGroup("ad3", List(realmGroup2Full.groupEmail)) { realmGroup3 =>
+        withGroup("ad3", List(realmGroup2Full.membersGroup.groupEmail)) { realmGroup3 =>
```

Comment:
why the changes in this file?

Reply:
There was a change to the `OrchestrationModel` at some point and `.groupEmail` is no longer a part of the `ManagedGroupWithMembers` class, it was moved to the `RawlsGroupShort` class. This change was needed for Rawls to compile when I updated the dependency to use a newer version of `serviceTest`. I'm now realizing that I had only updated the dependency for debugging purposes, so I can remove this change when I revert the `serviceTest` version to the older version.

Reply:
Actually I do need to update the `serviceTest` version, so I think I'll need to keep these changes.

Reply:
something doesn't seem right because we should not be using the membersGroup email for anything

Reply:
Okay, what email should be used then? The `ManagedGroupWithMembers` class doesn't have a group email associated with it anymore, and I am not sure what it should be replaced with.